Mm83 hero redesign

GitLukeW opened the pull request:
## **Pull Request Template**

### **1. Targeted Issue** ###

Ticket # 83

### **2. Overview of Solution** ###

Redesigned the hero on the homepage off of the approved design. Created two new components for the login button and the register button. Also starting to switch the files over to .jsx instead of .js. 


### **3. Tools Used** ###

Figma, MUI


### **4. Testing Strategy** ###

Tested the app to ensure all functionality working properly and no errors in the console. 


### **5. Future Implications** ###

Will need to update the nav bar next to match the design. 


### **6. Screenshots** ###

Before:
<img width="500" alt="Screenshot 2023-07-08 at 7 08 12 PM" src="https://github.com/TeamProductionSystem/Team_Production_System/assets/102833183/90bfb8be-8072-4283-8e34-1c7c5d0fbb6a">

After:
<img width="500" alt="Screenshot 2023-07-09 at 6 04 09 PM" src="https://github.com/TeamProductionSystem/Team_Production_System/assets/102833183/98363f75-9802-4952-a056-2ca5b61ec6f1">


### **7. Code Reviewers** ###

N/A


---

Before you go...

- [ ✓]  I have linked the relevant issue
- [✓ ]  I have provided an overview of my solution
- [ ✓]  I have shared the tools I've used
- [ ✓]  I have outlined my testing strategy
- [ ✓]  I have considered future implications of my changes
- [✓ ]  I have included relevant screenshots
- [✓ ]  I have kept my team in the loop

---

### Big high-fives for your incredible pull request! We're thrilled to have you as a contributor! Our team will dig into your proposed changes and give you some feedback or merge them if they're good to go. Your contributions mean the world to us!

nzeager reviewed (changes requested):
Love the new look! It looks great on the mobile view as well. A couple things before I merge:
1) The RegisterButton.jsx styling with marginLeft on line 29 doesn’t appear in LoginButton.jsx styling. Was that intentional?
2a) For the Register and Login components, there is a pacman loader button included, but it looks to me like it would display in addition to the button that is already there instead of replacing the button.
2b) When the Register and Login components are called in the Hero component the loading state is not passed in, so that logic doesn't currently do anything. (You'll also need to pass the loading state into the Hero component.)

GitLukeW commented:
> 

Great catch! I have updated the PR and removed the CSS from the components and added them into the parent component which is the hero component. Also took out the loading code per our conversation.

nzeager reviewed (approved):
Fixes look good!
